command-expansion-server/src/packages/db/db.ts

```diff
@@ -0,0 +1,40 @@
+import type { Pool, PoolConfig } from "pg";
+import pg from "pg";
+import { getEnv } from "../../env.js";
+
+const { Pool: DbPool } = pg;
+
+const {
+  POSTGRES_AERIE_EXPANSION_DB,
+  POSTGRES_HOST: host,
+  POSTGRES_PASSWORD: password,
+  POSTGRES_PORT: port,
+  POSTGRES_USER: user,
+} = getEnv();
+
+export class DbExpansion {
+  private static pool: Pool;
+
+  static getDb(): Pool {
+    return DbExpansion.pool;
+  }
+
+  static async init(): Promise<void> {
```

Comment:
I don't see any async in here right now. I assume we have this as an async method assuming that there is potential for it to be async in the future.

Reply:
Right now we are not doing anything async so I will remove it. If we do later, I can add it back in.